src/renderer/src/components/common/Text/Text.tsx

```
@@ -0,0 +1,51 @@
+}
+
+const Text = ({
+  children,
+  color,
+  fontType = 'H1',
+  width = 'auto',
+  textAlign = 'left',
+  ellipsis = false,
+  whiteSpace = 'nowrap',
+  tag = 'span'
+}: TextProps) => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)